src/ui/camera.js

```diff
@@ -342,6 +342,65 @@ class Camera extends Evented {
         return this;
     }
 
+    /**
+     * Pans and zooms the map to contain its visible area within the geographical bounds of the feature geometry.
+     * This function will also reset the map's bearing to 0 if bearing is nonzero.
```

Comment:
does this mean fit to feature is pitch aware and will retain the existing pitch?

Reply:
> This function will also reset the map's bearing to 0 if bearing is nonzero.

What are the implications of this for backwards compatibility? In the future if we can support retaining the bearing as part of `fitFeature` would it just be a breaking change?

Reply:
It does not alter the pitch, but it also doesn't account for it and will not give the correct result. 
There is no resolution yet for the pitch aware fitting.

There is more work needed to maintain the bearing or change it to some other value as part of the fitting operation.